subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/UpgradeDependencyPlugin.java

```diff
@@ -226,6 +228,26 @@ public void execute(Task task) {
         });
     }
 
+    private String getPullRequestDescription(CreatePullRequestTask task) {
+        if (StringUtil.isEmpty(task.getPullRequestDescription())) {
```

Comment:
This "if" is not needed here. Execute action will be executed right after creating a task so there is no way that user overrode one of the parameters in the meantime. User can still override it later.